apps/mi/mi-adf-shir-2/dev.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: mi-adf-shir-deployment-2
+  namespace: mi
+spec:
+  releaseName: mi-adf-shir-2
+  values:
+    image: sdshmctspublic.azurecr.io/mi/adf-integration-runtime:prod-552c219-20230403232719 #{"$imagepolicy": "flux-system:mi-adf-shir"}
+    replicaCount: 4
```

Comment:
there is only 1 cluster in dev, probbly is better to reduce the replica count to 2?